core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -283,6 +341,42 @@ oppia.controller('ExplorationEditor', [
       });
     };
 
+    $scope.showWelcomeExplorationModal = function() {
+      $scope.welcomeModalIsOpening = true;
```

Comment:
As far as I can tell, this variable is set but never used. Do you actually need it?